data/selinux/snappy.te

```diff
@@ -503,8 +503,14 @@ userdom_delete_user_tmp_files(snappy_mount_t)
 files_manage_generic_tmp_dirs(snappy_mount_t)
 files_manage_generic_tmp_files(snappy_mount_t)
 
-# Allow snap-{update,discard}-ns to mount on /etc/
-allow snappy_mount_t etc_t:dir mounton;
+# Allow snap-{update,discard}-ns to manipulate /etc/
+allow snappy_mount_t etc_t:dir { mounton write rmdir remove_name };
```

Comment:
See comment below... Does `etc_t:dir` require some sort of `gen_require` since you're allowing more operations on it now than before?

Reply:
`etc_t` already has a matcing `gen_require` which defines the type